src/core/GizmoViewport.tsx

```diff
@@ -109,7 +109,7 @@ export const GizmoViewport = ({
   hideAxisHeads,
   disabled,
   font = '18px Inter var, Arial, sans-serif',
-  axisColors = ['#ff3653', '#0adb50', '#2c8fdf'],
+  axisColors = ['#ff2060', '#20df80', '#2080ff'],
```

Comment:
Whats the reason for this change?

Reply:
i did this, the colors in gizmo were off, saturation wasn't accurate among the rgb pairs. but its almost imperceptible but good opportunity to clean up.